pkg/fieldpath/fieldpath_test.go

```diff
@@ -268,3 +284,60 @@ func TestSplitMaybeSubscriptedPath(t *testing.T) {
 		}
 	}
 }
+
+// TestFormatMap
+func TestFormatMap(t *testing.T) {
+	type args struct {
+		m map[string]string
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantFmtStr string
+	}{
+		{
+			name: "nil",
+			args: args{
+				m: nil,
+			},
+			wantFmtStr: "",
+		},
+		{
+			name: "label",
+			args: args{
+				m: map[string]string{
+					"beta.kubernetes.io/os":                 "linux",
+					"kubernetes.io/arch":                    "amd64",
+					"kubernetes.io/hostname":                "master01",
+					"kubernetes.io/os":                      "linux",
+					"node-role.kubernetes.io/control-plane": "true",
+					"node-role.kubernetes.io/master":        "true",
+				},
+			},
+			wantFmtStr: "beta.kubernetes.io/os=\"linux\"\nkubernetes.io/arch=\"amd64\"\nkubernetes.io/hostname=\"master01\"\nkubernetes.io/os=\"linux\"\nnode-role.kubernetes.io/control-plane=\"true\"\nnode-role.kubernetes.io/master=\"true\"",
+		},
+		{
+			name: "annotation",
+			args: args{
+				m: map[string]string{
+					"flannel.alpha.coreos.com/backend-data":                  "{\"VNI\":1,\"VtepMAC\":\"ce:f9:c7:a4:de:64\"}",
+					"flannel.alpha.coreos.com/backend-type":                  "vxlan",
+					"flannel.alpha.coreos.com/kube-subnet-manager":           "true",
+					"flannel.alpha.coreos.com/public-ip":                     "192.168.19.160",
+					"management.cattle.io/pod-limits":                        "{\"cpu\":\"11400m\",\"memory\":\"7965Mi\"}",
+					"management.cattle.io/pod-requests":                      "{\"cpu\":\"2482m\",\"memory\":\"7984Mi\",\"pods\":\"26\"}",
```

Comment:
use ` quoted strings to avoid excessive backslash escaping

Reply:
Fixed